job/server/src/main/java/alluxio/master/job/plan/PlanCoordinator.java

```diff
@@ -297,10 +296,13 @@ private synchronized void updateStatus() {
         // Try to join first, so that in case of failure we don't move to a completed state yet
         mPlanInfo.setResult(join(taskInfoList));
         mPlanInfo.setStatus(Status.COMPLETED);
+        LOG.info("Job completed Id={} Config={}", mPlanInfo.getId(), mPlanInfo.getJobConfig());
       } catch (Exception e) {
         mPlanInfo.setStatus(Status.FAILED);
         mPlanInfo.setErrorType(ErrorUtils.getErrorType(e));
         mPlanInfo.setErrorMessage(e.getMessage());
+        LOG.info("Job failed when joining tasks Id={} Config={} Error={}", mPlanInfo.getId(),
+            mPlanInfo.getJobConfig(), e.getMessage());
```

Comment:
If we print the error stack, it will give us more information to find the root cause.